resources/console/profile-evaluation.yaml

```diff
@@ -23,3 +23,11 @@ backend:
     limits:
       cpu: 50m
       memory: 80Mi
+pamela:
+  resources:
+    requests:
+      cpu: 8m
+      memory: 55Mi
+    limits:
+      cpu: 40m
+      memory: 150Mi
```

Comment:
Weekly and nightly clusters use 180Mi all the time, 150 might be not enough

Reply:
No, weekly uses 53MB normally, 63MB on peak...
![image](https://user-images.githubusercontent.com/22825433/103616027-f6ef5480-4f2b-11eb-99b4-9b1d16e5be16.png)

Reply:
You're right, I selected the wrong color on the graph, my bad. If it's 63, then 150 might be a bit too much. same with production profile. I think 200 would be enough there

Reply:
😕 so.. 200 or 150? I think `150Mi` would be perfect for the production profile and `100Mi` for the evaluation one. Do you agree?

Reply:
"150 might be a bit too much" was about the evaluation profile, "200 would be enough there" was about the production profile 🙂 sorry for the confusion.
Those values that you wrote should be fine 🙂